CacheSim/GenericHashTable.h

```diff
@@ -314,7 +378,7 @@ class GenericHashTable
     const size_t new_capacity = old_capacity ? old_capacity * 2 : 64;
 
     Elem** old_table = m_Table;
-    Elem** new_table = (Elem**) HeapAlloc(m_Heap, 0, sizeof(Elem*) * new_capacity);
+    Elem** new_table = (Elem**) VirtualMemoryAlloc(sizeof(Elem*) * new_capacity);
```

Comment:
Did you try this on change on Windows? I suspect substituting `VirtualAlloc` for `HeapAlloc` will make the code at least an order of magnitude slower. Capture speed for a large game is already pretty slow, so I'd want to be really careful here.